### DISCUSSION TITLE
"For You" needs to redesign

### DISCUSSION THREAD

**Discussion by torn4dom4n** [community/community#13649] [Other Feature Feedback, Questions, & Ideas]
This happens on my Samsung A50:

![Screenshot_20220328-012955_Chrome](https://user-images.githubusercontent.com/27698189/160295590-cc72957a-4e68-447b-8855-3660e1805c23.jpg)

You can see bad layout in the right.

**Comment by Dontmesswithtotal** [comment]
Ok

**Comment by jfullstackdev** [comment]
how about by now ? 

![image](https://user-images.githubusercontent.com/47092464/183014184-db94a333-7514-4d59-8111-dab06dc78737.png)

**Reply by torn4dom4n** [reply]
Now, it's better. Anyways, can `Send feedback` and `Filter` be improved?

![Screenshot_20220805-134014_Chrome](https://user-images.githubusercontent.com/27698189/183017689-46f16afc-13e7-4b10-b3fe-e16a565634bf.jpg)

**Reply by jfullstackdev** [reply]
that's mobile ? there is also similar support, the button is overlapping with the texts

**Reply by torn4dom4n** [reply]
Yes, my phone is Samsung A50.

**Reply by jfullstackdev** [reply]
let's wait for that
